Neos.FluidAdaptor/Classes/View/TemplatePaths.php

```diff
@@ -539,13 +539,14 @@ public function setOption($optionName, $value)
      * <PackageKey>_<SubPackageKey>_<ControllerName>_<prefix>_<SHA1>
      * The SH1 hash is a checksum that is based on the file path and last modification date
      *
-     * @param string $pathAndFilename
+     * @param string|null $pathAndFilename
      * @param string $prefix
      * @return string
      * @throws InvalidTemplateResourceException
      */
     protected function createIdentifierForFile($pathAndFilename, $prefix)
     {
+        $pathAndFilename = (string)$pathAndFilename;
```

Comment:
This is meh, but that's what TYPO3Fluid does too.

Reply:
o.O I wonder why they changed the param annotation then.. well

Reply:
🤷‍♂️